recipe/meta.yaml

```diff
@@ -1,20 +1,20 @@
 {% set name = "ensureconda" %}
-{% set version = "1.4.3" %}
+{% set version = "1.4.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e04ae2e2f872869df7e7da22dcca9bd7c42f839a55155dddb249bcdc9e6aae48
+  sha256: 2ee70b75f6aa67fca5b72bec514e66deb016792959763cbd48720cfe051a24a4
 
 build:
   number: 0
   skip: True  # [py<37]
   entry_points:
     - ensureconda = ensureconda.cli:ensureconda_cli
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
```

Comment:
`wheel` should be removed from `host` and hatchling should be `hatchling >=0.21.1`